AntennaTracker/Tracker.h

```diff
@@ -255,8 +253,10 @@ class Tracker : public AP_HAL::HAL::Callbacks {
     void start_logging();
     void log_init(void);
     bool should_log(uint32_t mask);
-    void calc_angle_error(float pitch, float yaw);
+    void calc_angle_error(float pitch, float yaw, bool direction_reversed);
     void calc_body_frame_target(float pitch, float yaw, float& bf_pitch, float& bf_yaw);
+    bool convert_bf_to_ef(float pitch, float yaw, float& ef_pitch, float& ef_yaw);
```

Comment:
because "calc_body_frame_target" and "convert_bf_to_ef" are very similar functions (one is the reverse of the other) I think it would be good to give them similar names.  so either rename "convert_bf_to_ef" to be "calc_earth_frame_target" (or something like that) or rename "calc_body_frame_target" to "convert_ef_to_bf".